src/main/resources/db/migration/0250/V281__Variables.sql

```diff
@@ -0,0 +1,145 @@
+CREATE TABLE methodologies (
```

Comment:
What would you think of concatenating all these migrations into one file? I don't think there's much benefit in having them separated out when we just care about getting to the right end state.

Reply:
Makes sense to me.